File: test/fixtures/multifile/android/src/main/java/com/example/multifile/MultiPlugin.java
package com.example.multifile;

import com.getcapacitor.JSObject;
import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "Multi")
public class MultiPlugin extends Plugin {
    @PluginMethod
    public void ping(PluginCall call) { call.resolve(new JSObject().put("value", true)); }
    @PluginMethod
    public void run(PluginCall call) { call.resolve(new JSObject().put("ok", true)); }
    @PluginMethod
    public void initialize(PluginCall call) { call.resolve(); }
}

File: test/fixtures/multifile/ios/Sources/MultiPlugin.swift
import Foundation
import Capacitor

@objc(MultiPlugin)
public class MultiPlugin: CAPPlugin, CAPBridgedPlugin {
    public let identifier = "MultiPlugin"
    public let jsName = "Multi"
    public let pluginMethods: [CAPPluginMethod] = []

    @objc func ping(_ call: CAPPluginCall) { call.resolve(["value": true]) }
    @objc func run(_ call: CAPPluginCall) { call.resolve(["ok": true]) }
    @objc func initialize(_ call: CAPPluginCall) { call.resolve() }
}
